app/src/dashboard/dashboard.html

```diff
@@ -48,11 +48,11 @@ <h1 class="md-display-2 text-center no-margin">{{ vm.dashboardData.shareAccounts
     <div layout="row">
         <md-card layout-gt-md="row" layout-align="center center" flex class="savings-total-card md-padding">
             <h1 class="display-1" flex>{{ 'label.heading.totalAccounts' | translate }}</h1>
-            <h1 class="display-1 text-right" flex="60">{{ vm.dashboardData.totalSavings }}</h1>
+            <h1 class="display-1 text-right" style="word-wrap:break-word;">{{ vm.dashboardData.totalSavings }}</h1>
         </md-card>
         <md-card layout-gt-md="row" layout-align="center center" flex class="loan-total-card md-padding">
             <h1 class="display-1" flex>{{ 'label.heading.totalLoan' | translate }}</h1>
-            <h1 class="display-1 text-right" flex="60">{{ vm.dashboardData.totalLoan }}</h1>
+            <h1 class="display-1 text-right" style="word-wrap:break-word;">{{ vm.dashboardData.totalLoan }}</h1>
```

Comment:
remove inline styles to the sass file, add a class name and I think it should be fine 👍

Reply:
@botraunak Is 'display-1' class is usable?

Reply:
no display-1 is a genric class a something like card-amount and you can find sass files in the assets/stylesheets directory where you can your class in custom.scss